contracts/optimistic-ethereum/OVM/chain/OVM_CanonicalTransactionChain.sol

```diff
@@ -327,14 +312,14 @@ contract OVM_CanonicalTransactionChain is iOVM_CanonicalTransactionChain, Lib_Ad
             timestampAndBlockNumber := or(timestampAndBlockNumber, shl(40, number()))
         }
 
-        iOVM_ChainStorageContainer queue = queue();
+        iOVM_ChainStorageContainer queueRef = queue();
 
-        queue.push2(
+        queueRef.push2(
             transactionHash,
             timestampAndBlockNumber
         );
 
-        uint256 queueIndex = queue.length() / 2;
+        uint256 queueIndex = queueRef.length() / 2;
```

Comment:
would it be more accurate used
`uint256 queueIndex = queueRef.length() / 2 - 1`, 
rather than doing the subtraction below?

Reply:
@smartcontracts going to make this change unless there's any objection :)

Reply:
No objection.